src/checks/checkVersionInfo.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  basicJsonFileValidator,
+  decodeJson,
+  parseJson,
```

Comment:
Since they are not used, they could be removed
```suggestion
```

Reply:
tnx, I totally missed all the unused import. Fixed in 3dd29ac but also created https://github.com/pagopa/io-services-metadata/pull/555 with the latest ESLint version in order to avoid the same error again.